apps/storybook/stories/icons/overview.mdx

```diff
@@ -41,3 +64,54 @@ export const App = () => {
 };
 `}  dark/>
 
+
+
+<Subtitle>Usage without foundation package</Subtitle>
+Icons are completely consumable without foundation package. I want to highlight some of th key points and simplifications that are made for that.
+
+Bu default main and accent parts of the svgs are `fill=currentColor` so in most cases you don't need to do anything.
```

Comment:
By default main and accent parts of the svgs are `fill=currentColor` so in most cases you don't need to do anything.
```suggestion
Bu default main and accent parts of the svgs are `fill=currentColor` so in most cases you don't need to do anything.
```